src/card.css

```diff
@@ -0,0 +1,99 @@
+.card {
+  width: 200px;
+  background-color: var(--color-white);
+  border: 1px solid var(--color-concrete);
+}
+  
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin-left: calc(50% - 160px / 2);
+  margin-top: 33px;
+  margin-bottom: 36px;
+}
+  
+.card__stars_box {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  margin-right: calc(50% - 166px / 2);
+  margin-left: calc(50% - 166px / 2);
```

Comment:
Let's specify correct paddings for the card class instead of adding margin-left / margin-right for each element

Reply:
i have to use margin: 0 calc(50% - 166px / 2); or dont use this formuls?